resources/netlog_defs.ts

```diff
@@ -1014,17 +1024,41 @@ const latestLogDefinitions = {
       timestamp: 1,
       change: 2,
       id: 3,
-      // from here, pairs of field name/values, e.g.
-      key1: 4,
-      value1: 5,
-      // key2: 6,
-      // value2: 7,
+      // from here, pairs of field name/values
     },
     canAnonymize: true,
     firstOptionalField: 4,
     playerIds: {
       3: null,
     },
+    repeatingFields: {
+      startingIndex: 4,
+      label: 'pair',
+      names: ['key', 'value'],
+      sortFn: (left: unknown, right: unknown): number => {
```

Comment:
Sorry to restate this, but just trying to reiterate all the open issues on my mind.

Looking for these changes:
* `sortFn: (left: string, right: string): number` as a type in the definition
* ideally then `sortFn: (left: string, right: string) => left.localeCompare(right),` here?
* this helper code to go from unknown => string in the caller in netregexes.ts / regexes.ts (maybe some shared helper function somewhere) which itself can call the string `sortFn` at the bottom

That way future `sortFn` can just write a very trivial sort function rather than having to re-do this helper goop eveyrwhere.

Reply:
Hmm, ok.  I guess I assumed we'd always have at least some string keys to sort on here for anything we'd want to match against, but maybe that's not true.  I guess we only have one example so I shouldn't try to oversimplify yet.